### DISCUSSION TITLE
I can not find BeautifulSoup==3.0.7a

### DISCUSSION THREAD

**Discussion by mhachemi1403** [mediadrop/mediadrop#258] [General]
Hi everyone;
I can't find the BeautifulSoup == 3.0.7a library and it is required by Mediacore to run the mediadrop application.
Please can someone help to resolve this problem.

Thank you in advice.

**Comment by FelixSchwarz** [comment]
Try:

- `pip install -r requirements.txt`

or

- `pip install --index-url=https://static.mediadrop.video/dependencies/dev/ 'BeautifulSoup==3.0.7a'`
